deploy/inginious-base.containerfile

```diff
@@ -0,0 +1,15 @@
+FROM rockylinux:8
```

Comment:
Is this the ` ${REGISTRY}/inginious/core-base` container ? If so, maybe it's worth documenting it somewhere, maybe simply with a comment header here.

Reply:
Solved in 3a4822341347e0e649310cc9f52201d4c9637115.